### DISCUSSION TITLE
I seem to have a problem with Prisma (/w blitz@latest)...

### DISCUSSION THREAD

**Discussion by AxDSan** [blitz-js/blitz#1288] [General] [answered]
Should I open the issue here or report directly at the prisma repo?

**Comment by flybayer** [comment, marked as answer]
If it's prisma specific, yeah ask them. You can ask in [their slack](https://prisma.slack.com).

**Comment by AxDSan** [comment]
Thanks Brandon, I just also [commented this](https://blitzjs.slack.com/archives/C011EQ0J75M/p1602367706035600?thread_ts=1602302363.004900&cid=C011EQ0J75M) on the Blitz slack, and I was able to fix this issue :D thanks for replying :D
